### ISSUE TITLE
IBM Accessibility Checker is giving many violations for TinyMCE editor

### ISSUE THREAD

**Issue by kishoreBabuPosa** [tinymce/tinymce#8425]
Hi, I am using TinyMCE editor in my application, while running the IBM accessibility checker getting many violations and warnings. Please address them if possible. Please have some of the violations for your reference.
1)None of the descendent elements with "menuitem" role is tabbable.
2)Element with "application" role does not have a label
3)The element with "complementary" role does not have a label
4)Accessible name does not match or contain the visible label text
5)Multiple toolbar components do not have unique labels
